src/lib.rs

```diff
@@ -37,19 +131,17 @@ extern crate flate2;
 extern crate rand;
 extern crate x86intrin;
 
-// TODO: don't expose everything!
 #[macro_use]
 pub mod errors;
 pub mod basic;
 pub mod data_type;
 mod parquet_thrift;
 
 #[macro_use]
-pub mod util;
```

Comment:
This will break `cargo bench`. We need to find a way to expose only the necessary APIs.